src/control.ts

```diff
@@ -73,6 +74,10 @@ export function dopriControl(control: Partial<DopriControlParam> = {}) {
         tcrit: withDefault(control.tcrit, defaults.tcrit),
     };
 
+    if (ret.tcrit.length > 0) {
+        ret.tcrit = [...ret.tcrit].sort();
```

Comment:
also not sure why there is a copy here?
```suggestion
        ret.tcrit.sort();
```

Reply:
This copy is required I believe, otherwise if the user passes in some array, `sort` mutates it for the parent, which is gross. This is like having some pure function `sorted()` with which we could have written

```
ret.tcrit = sorted(ret.tcrit)
```